src/static/css/2019.css

```diff
@@ -49,6 +49,31 @@ select {
   color: white;
 }
 
+a.navigation-logo {
+  outline: none;
+  color: white;
+}
+a.navigation-logo img:hover {
+  filter: brightness(0) saturate(100%) invert(86%) sepia(4%) saturate(4357%) hue-rotate(16deg) brightness(136%) contrast(94%);
+}
+a.navigation-logo:focus img {
+  filter: brightness(0) saturate(100%) invert(86%) sepia(4%) saturate(4357%) hue-rotate(16deg) brightness(136%) contrast(94%);
+}
+
+header.alt-bg a.interactive:hover, footer.alt-bg a.interactive:hover,
```

Comment:
Why is `a.interactive:hover` needed and not simply `a:hover`? Are there some anchors that shouldn't inherit these interactive styles?

Reply:
Certainly, this is simple text a hover/focus, so this useless selector (`interactive`) can be removed.
I agree and fix it.